tests/test_zero/test_gemini/test_zeroddp_state_dict_shard.py

```diff
@@ -30,15 +30,16 @@ def exam_state_dict(placement_policy, model_name: str):
 
     zero_dict = model.state_dict(only_rank_0=False)
     accumulated_keys = set()
+    total_size = 0
     # ensure number of shards > 1
-    for shard in model.state_dict_shard(max_shard_size=(model_size / 3), only_rank_0=False):
+    for shard, s_size in model.state_dict_shard(max_shard_size=(model_size / 3), only_rank_0=False):
+        total_size = total_size + s_size
```

Comment:
Do you need to check the total size? If not, I think we can delete this variable.

Reply:
ok